pkg/vsphere/tasks/waiter.go

```diff
@@ -164,11 +201,11 @@ func logFault(op trace.Operation, fault types.BaseMethodFault) {
 }
 
 func logSoapFault(op trace.Operation, fault types.AnyType) {
-	op.Debugf("unexpected soap fault on task retry : %#v", fault)
+	op.Debugf("unexpected soap fault on task retry : %s", fault)
```

Comment:
It was dumping the struct but not showing the error (instead it is showing the address of the err field)